iOS/Extensions/UserDefaults/UserDefaults+Theme.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  Created for xikolo-ios under MIT license.
+//  Copyright © HPI. All rights reserved.
+//
+
+import Common
+import Foundation
+
+@available(iOS 13.0, *)
+enum Theme: Int, CaseIterable {
+    case device
+    case light
+    case dark
+
+    static var orderedValues: [Theme] {
+        return [.device, .light, .dark]
+    }
```

Comment:
There need to add this as it will be provided automatically when conforming to `CaseIterable` (as `allCases`) ;) 

We have those `orderedValues` implementations in our codebase because the functionality was required before `CaseIterable` was introduced. They still exist to avoid potential side effects.